desmeds/desdm_maker.py

```diff
@@ -977,38 +972,24 @@ def get_rec(self, row, col):
         image is normalized
         """
 
-        y, x, = self._get_yx(row, col)
-
         gsim = self.piff_obj.draw(
-            x=x,
-            y=y,
+            x=col,
+            y=row,
+            center=True,
```

Comment:
We should double-check that this overrides properly. I think it does.

Reply:
Sorry, what overrides?

Reply:
```
4. If you want the PSf centered exactly in the center of the image, then you can use
   ``center=True``.  This will work for either an automatically built image or one
   that you provide.
```

Reply:
Yeah this should be fine. You are right.